### PR TITLE
tests: add spread test for snap create-key/snap sign

### PR DESCRIPTION
**Pull request by mvo5**

### CODE REVIEW

**Comment by zyga** [comment]
Looks good but it doesn't seem to work yet

File: tests/main/snap-sign/task.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+summary: Run snap sign to sign a model assertion
+
+execute: |
+    echo "Creating a new key without a password"
+    expect -f successful_default.exp
```
Comment (chipaca) [inline tests/main/snap-sign/task.yaml]: I believe this should be either a copy of, or point to `../create-key/successful_default.exp`

File: tests/main/snap-sign/create-key-no-pw.exp [hunk 2]
```diff
@@ -0,0 +1,15 @@
+spawn snap create-key
```
Comment (chipaca) [inline tests/main/snap-sign/create-key-no-pw.exp]: what runs this?

Comment (zyga) [inline tests/main/snap-sign/create-key-no-pw.exp]: Yep, this seems unused in this PR.

**Comment by niemeyer** [comment]
Needs some love.

**Pull request merged**